### PR TITLE
ROX-19328 - Add gitops validate CLI command

### PR DESCRIPTION
**Pull request by SimonBaeumer**
## Description
[ROX-19328](https://issues.redhat.com//browse/ROX-19328) - Add gitops validate CLI command

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Added test description under `Test manual`
- [ ] Documentation added if necessary (i.e. changes to dev setup, test execution, ...)
- [ ] CI and all relevant tests are passing
- [ ] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.
- [ ] Add secret to app-interface Vault or Secrets Manager if necessary
- [ ] RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)
- [ ] Check AWS limits are reasonable for changes provisioning new resources

## Test manual

 - Use acsfleetctl to validate config files in gitops repo

### CODE REVIEW

File: internal/dinosaur/pkg/gitops/cmd/validate.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+			}
+
+			fieldPath := &field.Path{}
+			errors := operator.Validate(fieldPath, configs)
+			if len(errors) > 0 {
+				glog.Errorf("gitops validation failed")
+				for _, err := range errors {
+					glog.Error(err)
+				}
+				os.Exit(1)
```
Comment (kurlov) [inline internal/dinosaur/pkg/gitops/cmd/validate.go:53]: Why there is `os.Exit(1)` instead of `return` like in reading the file?

Comment (SimonBaeumer) [reply internal/dinosaur/pkg/gitops/cmd/validate.go:53]: good catch, fixed it

**Pull request merged**